fn main() {
    let num_err = 69;
    eprintln!("This program ran into {num_err} errors.");
    eprintln!("This program ran into {} errors.", num_err);
}
